v22.1/dbeaver.md

```diff
@@ -30,43 +30,52 @@ To work through this tutorial, take the following steps:
 
 ## Step 1. Start DBeaver and connect to CockroachDB
 
-Start DBeaver, and select **Database > New Connection** from the menu.  In the dialog that appears, select **CockroachDB** from the list.
+1. Start DBeaver, and select **Database > New Connection** from the menu.  In the dialog that appears, select **CockroachDB** from the list.
 
-<img src="{{ 'images/v2.1/dbeaver-01-select-cockroachdb.png' | relative_url }}" alt="DBeaver - Select CockroachDB" style="border:1px solid #eee;max-width:100%" />
+    <img src="{{ 'images/v22.1/dbeaver-01-select-cockroachdb.png' | relative_url }}" alt="DBeaver - Select CockroachDB" style="border:1px solid #eee;max-width:100%" />
 
-## Step 2. Update the connection settings
+1. Click **Next**. The **Connection Settings** dialog displays.
 
-On the **Create new connection** dialog that appears, click **Network settings**.
+1. In the **Database** field, enter **movr**.
```

Comment:
Should "movr" actually be in code ticks, since it is technically a value that we are entering? Not sure if our style is settled on this.